code/iaas/allocator/src/main/java/io/cattle/platform/allocator/constraint/ContainerLabelAffinityConstraint.java

```diff
@@ -10,7 +10,7 @@
 import com.google.common.collect.Multimap;
 
 public class ContainerLabelAffinityConstraint implements Constraint {
-    public static final String ENV_HEADER_AFFINITY_CONTAINER_LABEL = "affinity:container_label:";
+    public static final String ENV_HEADER_AFFINITY_CONTAINER_LABEL = "affinity:";
```

Comment:
No.  Although, I really wonder how many people use this since we encourage using labels instead of environment variables.  @ibuildthecloud What do you think?  Basically, we introduced our syntax before swarm.  This PR is to keep parity with swarm.